src/utils/__fixtures__/test-inputs.ts

```diff
@@ -426,4 +448,4 @@ export const walletExtensionV0UnlockedState = {
     }),
     enteredWrongPassword: false,
   },
-} satisfies RootState
+} as RootState
```

Comment:
This reduces type safety

Reply:
It does, but refers to my question here https://github.com/oasisprotocol/oasis-wallet-web/pull/1954#discussion_r1645389582

How come the migration now adds nonce to wallet/s, the only change I made was to the account interface? And saved the nonce from account fetch to the account object. Do you maybe know from the top of your head? Before I make a deep dive.

Reply:
walletExtensionV0UnlockedState: migrating-v0-ext-persisted-state.spec.ts is fetching real data https://github.com/oasisprotocol/oasis-wallet-web/issues/1770

privateKeyUnlockedState: it is fetching mocked data

Reply:
Oh - why is it in wallet/s. I'm not entirely sure, but related to https://github.com/oasisprotocol/oasis-wallet-web/commit/abe01239ff358026b2c6df2c5160cb69c9a5afe8. We aren't removing extra fields when passing balances between account and wallet: https://github.com/oasisprotocol/oasis-wallet-web/blob/8ca2795a923a3469d0f69f371f84535e2cf971f9/src/app/state/wallet/saga.ts#L72

Reply:
I am not sure why we even store balances in the wallet/s? As source of truth is on the blockchain, and values may not be correct.

Reply:
Just legacy
After loading stored wallets we update their balances through selectNetwork -> refreshOnNetworkChange -> fetchWallet